docker-for-mac/index.md

```diff
@@ -147,7 +147,11 @@ You can also move the disk image to a different location. If you attempt to move
 
 #### File sharing
 
-Choose the local directories you'd like to share with your containers. File sharing is required for volume mounting if the project lives outside of the `/Users` directory. In that case, share the drive where the Dockerfile and volume are located. Otherwise, you get `file not found` or `cannot start service` errors at runtime`.
+Use File sharing to allow local directories on the Mac to be shared with Linux containers.
+This is especially useful for
+editing source code in an IDE on the host while running and testing the code in a container.
+By default the `/Users` directory is shared. If your project is outside this directory then it must be added
```

Comment:
I think we share a few more by defaults, like the folders which have temp directories?

Reply:
This PR no longer includes the Mac stuff.